pkg/volume/rbd/rbd_util.go

```diff
@@ -264,7 +264,7 @@ func (util *RBDUtil) AttachDisk(b rbdMounter) error {
 		// modprobe
 		_, err = b.plugin.execCommand("modprobe", []string{"rbd"})
 		if err != nil {
-			return fmt.Errorf("rbd: failed to modprobe rbd error:%v", err)
+			glog.Warningf("rbd: failed to load rbd kernel module:%v", err)
```

Comment:
The issue with checking is that if Kubelet runs as a self-hosted or in a container, it wouldnt be able to do any modprobe operation. Ideally, this plugin should not be responsible for loading the kernel module. This is usually a one-time operation and done by the admin that setups the hosts. Even If the module is not loaded, we will get an error just a few lines after when it tries to attach the volume.